_posts/2017-03-31-introducing-deckgl-v4.md

```diff
@@ -110,35 +109,31 @@ deck.gl v3 came with essentially a single example that was interleaved with the
 
 ## Interoperability Examples
 
-To leverage your knowledge in data visualization with the tools you are familiar with, we also provide examples to demonstrate the interoperability between deck.gl and regular SVG-based visualizations. That is, you can use for example the modularized d3-* utilities for layout calculation, and have deck.gl take care of the rendering, so there is no heavy DOM manipulations and you get GPU-accelerated rendering of up to millions of elements for free.
+To leverage your knowledge in data visualization with the tools you are familiar with, we also provide [examples to demonstrate the interoperability](https://github.com/uber/deck.gl/tree/master/examples/svg-interoperability) between deck.gl and regular SVG-based visualizations. That is, you can use, for example, the modularized d3-* utilities for layout calculation, and have deck.gl take care of the rendering, so there is no heavy DOM manipulations and you get GPU-accelerated rendering of up to millions of elements for free.
 
 ## What’s Next?
 
-deck.gl v4 is an important milestone for us, but what's next? While we haven't finalized
-any plans for a possible deck.gl v4.1 or v5 release yet, we have many ideas about
-improvements and directions we would like deck.gl to evolve along. Some examples are:
+deck.gl v4 is an important milestone for us, but what's next? While we haven't finalized any plans for a possible deck.gl v4.1 or v5 release yet, we have many ideas about improvements and directions we would like deck.gl to evolve along. Some examples are:
 
 ### More and Better Layers
 
-Layers are how deck.gl creates value for most users, so obviously we plan to add more layers. We will of course always keep improving support for the geospatial vusalizations but we are also
-considering supporting abstract visualizations in the InfoVis area.
+Layers are how deck.gl creates value for most users, so adding more layers is on the top of our list. We will always keep improving support for the geospatial vusalizations while also considering supporting abstract visualizations in the InfoVis area.
```

Comment:
Is `InfoVis` an obvious term, especially with this distinctive capitalization? 

As it is suddenly mentioned, should there be a link to wikipedia or similar? 

@gnavvy @ericsoco